frontend/package.json

```diff
@@ -21,5 +22,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cypress": "^13.13.0"
```

Comment:
I'd install Cypress at the project's root instead of the `frontend/` directory since it tests both the frontend and backend.

Reply:
Good catch, this probably remained there after I installed cypress at the frontend level. I have now corrected this problem, thanks Walmyr 🥇